using-timescaledb/backup.md

```diff
@@ -331,6 +328,26 @@ end as the recovery will be complete when no further files can be
 found in the archive. See the PostgreSQL documentation on
 [continuous archiving][pg archiving] for more information.
 
+### Restore points [](restore_points)
+
+TimescaleDB natively support a restore point both for single-node and for
+multi-node environments. The restore point can later be used as a recovery target on
+single-node or on each node of the entire multi-node cluster, ensuring that it can be
+restored to a consistent state.
+
+To create a restore point on a single-node [`pg_create_restore_point`][pg_create_restore_point]
+function can be used.
+
+On a multi-node node [`create_distributed_restore_point`][create_distributed_restore_point] function must be used
+instead and executed on the access-node. In order to achieve synchronization across the multi-node cluster, this function ensures
+that the restore point created on the access node is synchronized with each data node. Care should be taken to ensure
+third-party solutions properly quiesce the environment before backing up, so that the
+restore point used across nodes does not have outstanding transactions running on the data nodes.
+
+Before using restore point make sure PostgreSQL is configured for [WAL Archiving][wal archiving]. To recover to the desired
```

Comment:
I think this paragraph is confusing.  It points to a specific section of postgres doc, where instead I think it should also apply to higher-level tools like pgBackrest or WAL-G that use WAL-archiving, no?  I probably would clarify this.

Also, the language could be clearer:

```In order to use restore points, you must configure PostgreSQL to use [WAL archiving][wal archiving], such
that the database stores a copy of all WAL segment, where the restore point will point to a specific
segment (location) in the WAL in which to restore to.  To use a restore point, ...
```
maybe you can finish the paragraph.  It could refer to the same language you currently use, but I think there are probably higher-level pointers to tooks like pgbackrest or WAL-G?

Reply:
I guess part of this section was related to the data nodes synchronization, mentioned above. I've tried to reformulate the paragraph and reference the third-party tools instead of pointing in to the PG doc